### ISSUE TITLE
icons and color used

### ISSUE THREAD

**Issue by sflavier** [RefLex-Team/RefLex-issues#17]
During the Adamawa workshop, some feedback were on the lacking of accessibility of the website for color vision deficiency.

I will try to keep track of the problems and explain the change made on the website.
